packages/measure/src/measure-renders.tsx

```diff
@@ -35,17 +35,28 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
   const warmupRuns = options?.warmupRuns ?? config.warmupRuns;
 
   const { render, cleanup } = resolveTestingLibrary();
+  const testingLibrary = getTestingLibrary();
 
   showFlagsOutputIfNeeded();
 
   const runResults: RunResult[] = [];
   let hasTooLateRender = false;
+  let hasUnnecessaryRender = false;
+  let components: RenderMeasureResult = [];
```

Comment:
The types does not seem correct.

```suggestion
type ToJsonTree = ReactTestRendererJSON | ReactTestRendererJSON[] | null;

let renderJsonTrees: ToJsonTree = [];
```


<img width="694" alt="image" src="https://github.com/callstack/reassure/assets/6368606/9b4af8f0-cf06-49fd-90de-3db7105ac36a">